src/ServiceProvider.php

```diff
@@ -95,6 +97,48 @@ protected function registerPermissions()
         }
     }
 
+    protected function registerSearchables()
+    {
+        Searchables::register('runway', function ($resource, $config) {
```

Comment:
Another note: I've refactored this code to index the fields specified in the search config & I believe I've also made transformers work (I haven't tested them yet).

I've also refactored it to loop via a `Collection` instead of 'foreaching' through resources.